tests/integration/targets/acme_letsencrypt/runme.sh

```diff
@@ -3,4 +3,5 @@
 set -eux
 
 ansible-playbook dns-challenge-pebble.yml
+ansible-playbook dns-challenge-hetzner.yml -e acme_hetzner_auth_token=$HETZNER_AUTH_TOKEN
```

Comment:
Hm, the actions does not seem to recognize the extra var 🤔 Is it set correctly?

Reply:
Yes, it is. I'm trying to find out how to pass it to ansible-test. It's not that easy it seems: https://docs.ansible.com/ansible/latest/dev_guide/testing_running_locally.html#environment-variables

Reply:
It's basically impossible to do securely: https://stackoverflow.com/a/67587866/2953919

Reply:
how about you set it as an environment variable and then add a task to read this via https://docs.ansible.com/ansible/latest/collections/ansible/builtin/env_lookup.html?